ably/doc.go

```diff
@@ -0,0 +1,29 @@
+// Package ably is the official Ably client library for Go.
+//
+// Get started at https://ably.io.
```

Comment:
The main website doesn't seem like a good place to send someone to get started with using the Go client, is there not a Go specific guide we can link to?

Reply:
I think `https://www.ably.io/documentation` would suffice.

That page prominently displays the [Quickstart Guide](https://www.ably.io/documentation/quick-start-guide). Though, looking at that guide, there's no code examples in Go so that's something that'll need addressing.

Reply:
Yup, we already have a ticket for that. See https://github.com/ably/docs/issues/572

I'll reassign that ticket to @tcard for now.

Reply:
Ok, so in the meantime, perhaps a link to the README in GitHub would be more useful? For example:

https://github.com/ably/ably-go#using-the-realtime-api

I highly suspect that anyone reading this doc will already have an idea about how Ably works and just wants some examples to get going with some Go code.

Reply:
Done at 0b238c2, thanks.